specification/netapp/resource-manager/Microsoft.NetApp/stable/2019-10-01/netapp.json

```diff
@@ -0,0 +1,2695 @@
+{
```

Comment:
For API reviews, we like to look at diffs between the last reviewed version (the most direct ancestor to the new version). To simplify this (in the future), please make a commit with the unmodified prior version, and then make the changes in a subsequent commit. This enables diff in the PR.

Reply:
Hmmm, I have performed an initial commit with the previous API and it is listed with the commits above immediately below the description https://github.com/Azure/azure-rest-api-specs/pull/7908/commits/ba1684a7bfd69c33fe61a9e28df3d31b8dda95cc. Is this not sufficient?